docs/adapters/configuration.md

```diff
@@ -0,0 +1,105 @@
+## Goal
+Most simply and consistently across adapters, allow users to configure one or multiple connections in addition to a single path to the adapter module. 
+
+## Options and Problems
+
+### 1. Single connection object.
+- Eval: Simple, can include `path`, but does not allow for several connections/environments (e.g. prod/staging/dev).
```

Comment:
`path` tells us where the module is. Doesn't have to do with the connection information. It is also `one` path per adapter where connection is `n` per adapter.